### ISSUE TITLE
No option to use delimiters while listing blobs

### ISSUE THREAD

**Issue by rachari** [Azure/azure-xplat-cli#3158]
The cli command to list storage blobs has no option to use a delimiter

Usage: storage blob list [options] [container] [prefix]

Under options, nothing is mentioned about delimiters and the help also doesn't state if there's any way to use it.

The Python SDK for Azure, specifically the list_blobs method has an option for specifying the delimiter.

Can this option be added to the azure-cli

**Comment by amarzavery** [comment]
@yaxia @hasonmsft - Can you respond to this issue?

**Comment by hasonmsft** [comment]
@rachari , delimiters are used for "virtual" directory for blobs. As currently we only have the commands to list raw blobs so that the delimiters are not supported. 

Mau I know what's your intention to use delimiter? To list "virtual" directories of the container?

**Comment by rachari** [comment]
@hasonmsft , Yes. My intention is to list the virtual directories of the containers using the delimiters

**Comment by hasonmsft** [comment]
@rachari , we'll put this request into our backlog. For now, you can use [Storage Explorer](http://storageexplorer.com/) to view the virtual directories.

**Comment by rachari** [comment]
@hasonmsft , Thanks for the update. 
I have been using the Storage Explorer for viewing the virtual directories. 
I've also tried out the list_blobs method with delimiters from the Python SDK for Azure which does work for listing out virtual directories with command line tools / scripts.
